web/custom-plugins/forecast-of-response/src/components/App/AppHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { TopBar } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import Icon from '../Icons'
+import { TAppHeader } from '../../Types'
+
+const Icons = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+  > * {
+    margin-left: 40px;
+  }
+`
+
+export default (props: TAppHeader): JSX.Element => {
```

Comment:
Not a fan of everything being called "AppHeader", "AppContent" etc.
Makes me wonder if there are other more "important" 'Header' components.